### PR TITLE
fix select text

### PR DESCRIPTION
**Pull request by snopoke**
@orangejenny

### CODE REVIEW

**Comment by snopoke** [comment]
no test so killed the build

File: corehq/apps/style/static/hqstyle/js/jquery/plugins/jquery.selectText.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-    var obj = this[0];
-    if ($.browser.msie) {
-        var range = obj.offsetParent.createTextRange();
-        range.moveToElementText(obj);
+// http://stackoverflow.com/a/987376
+jQuery.fn.selectText = function(){
+    var doc = document
+        , element = this[0]
+        , range, selection
+    ;
```
Comment (dimagimon) [inline corehq/apps/style/static/hqstyle/js/jquery/plugins/jquery.selectText.js]: Expected indentation of 8 space characters but found 4. (indent)

File: corehq/apps/style/static/hqstyle/js/jquery/plugins/jquery.selectText.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
-jQuery.fn.selText = function() {
-    var obj = this[0];
-    if ($.browser.msie) {
-        var range = obj.offsetParent.createTextRange();
-        range.moveToElementText(obj);
+// http://stackoverflow.com/a/987376
+jQuery.fn.selectText = function(){
```
Comment (dimagimon) [inline corehq/apps/style/static/hqstyle/js/jquery/plugins/jquery.selectText.js]: 'jQuery' is not defined. (no-undef)

**Comment by orangejenny** [comment]
nice

**Pull request merged**